lib/Graph/Graph.cpp

```diff
@@ -639,6 +641,30 @@ FullyConnectedNode *Function::createFullyConnected(llvm::StringRef name,
   return addNode(new FullyConnectedNode(name, OT, input, W, B));
 }
 
+RowwiseQuantizedFullyConnectedNode *
+Function::createRowwiseQuantizedFullyConnected(Context &ctx,
+                                               llvm::StringRef name,
+                                               NodeValue input, Storage *W,
+                                               Node *B, TypeRef outTy) {
+  // Since W is constant, quantize it in compilation time.
+  // The quantized data is in qWeights, the scale of each row is in scales,
+  // and the offset of each row is in offsets.
+  Placeholder *weights = llvm::cast<Placeholder>(W);
```

Comment:
this would be a constant weight and we'd need to fix that later

Reply:
If I understand correctly here I think we want to use `Variable`, not `Placeholder`, since the intent is that the RowwiseQuantizedFC is given constant weights.  Right?

If it's the case that we need constant weights here, can we make the function signature take a `Variable *` instead of a `Storage *`, so that the API is type safe?

Reply:
"If I understand correctly here I think we want to use Variable, not Placeholder, since the intent is that the RowwiseQuantizedFC is given constant weights. Right?" 
--- Yes, the weights should be constant.
So in the future, "Variable" means "constant" ?